tests/Oryx.BuildImage.Tests/Node/NodeJSSampleAppsTestBase.cs

```diff
@@ -24,5 +28,43 @@ public class NodeJSSampleAppsTestBase : SampleAppsTestBase
             }))
         {
         }
+
+        [Fact]
+        public void GeneratesScript_AndLoggerFormatCheck()
+        {
+            // Arrange
+            var volume = CreateWebFrontEndVolume();
+            var appDir = volume.ContainerDir;
+            var appOutputDir = "/tmp/" + SampleAppName + "-output";
+            var script = new ShellScriptBuilder()
+                .AddDefaultTestEnvironmentVariables()
+                .AddCommand($"echo RandomText >> {appDir}/Program.cs") // triggers a failure
+                .AddBuildCommand(
+                $"{appDir} -o {appOutputDir}")
+                .ToString();
+            // Regex will match:
+            // "yyyy-mm-dd hh:mm:ss"|ERROR|Micro
+            Regex regex = new Regex(@"""[0-9]{4}-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0-1]) (0[0-9]|1[0-9]|2[0-3]):([0-5][0-9]):([0-5][0-9])""\|ERROR\|.*");
```

Comment:
Could we fine tune this regex to pattern match a more granular message. Similar to: https://github.com/microsoft/Oryx/blob/main/tests/Oryx.BuildImage.Tests/golang/GolangDynamicInstallationTest.cs#L91

Reply:
I made the warning message as specific as possible.